exercises/concept/meltdown-mitigation/results.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
Should this file be committed in this PR? Suggest deletion of same.

Reply:
Ooof!  No.  This is a results file from testing and should not be here.  Thank you for catching that!!